FreeAgent.postman_collection.json

```diff
@@ -1152,6 +1152,160 @@
 				}
 			]
 		},
+		{
+			"name": "Bill Items",
+			"item": [
+				{
+					"name": "index",
+					"request": {
+						"auth": {
+							"type": "oauth2",
+							"oauth2": [
+								{
+									"key": "useBrowser",
+									"value": false,
+									"type": "boolean"
+								},
```

Comment:
I don't think this section is needed, it seems to create a new authentication scheme just for these new actions rather than inheriting the configured authentication scheme on the FreeAgent collection.

It's showing up as this:

![image](https://user-images.githubusercontent.com/482089/105707520-fc2c3780-5f0a-11eb-9155-a51303c5adb3.png)

But should be:

![image](https://user-images.githubusercontent.com/482089/105707556-08b09000-5f0b-11eb-8b6d-8b3fa1132711.png)

Maybe the setup for this repo has changed, but the client secret/id variables are no longer defined so this doesn't work out the box. Inheriting auth works fine though.

Reply:
I've made these auth all inherit from parent